packages/cli/src/modules/mcp/mcp.settings.controller.ts

```diff
@@ -22,18 +21,14 @@ export class McpSettingsController {
 	}
 
 	@Patch('/settings')
-	async updateSettings(req: AuthenticatedRequest) {
+	async updateSettings(req: AuthenticatedRequest, _res: Response, @Body dto: UpdateMcpSettingsDto) {
```

Comment:
I still think this model is a bit weird that we provide something that user can enable/disable from the UI and that is loaded from a module. IMO the configuration part should be outside the module, and the module should then be loaded based on the user configuration. But this is something that can be done later on